scripts/bert/finetune_classifier.py

```diff
@@ -344,9 +344,9 @@ def train(metric):
                     metric_val = [metric_val]
                 eval_str = '[Epoch {} Batch {}/{}] loss={:.4f}, lr={:.7f}, metrics=' + \
                     ','.join([i + ':{:.4f}' for i in metric_nm])
-                logging.info(eval_str.format(epoch_id + 1, batch_id + 1, len(train_data), \
-                    step_loss / args.log_interval, \
-                    trainer.learning_rate, *metric_val))
+                logging.info(eval_str.format(epoch_id + 1, batch_id + 1, len(train_data),
+                                             step_loss / args.log_interval,
+                                             trainer.learning_rate, *metric_val))
```

Comment:
nit: logging allows lazy construction of the string. you can make use of that by doing
`logging.info(eval_str, epoch_id + 1, batch_id + 1, ...)`. As a result, the logging string is not actually constructed unless logging actually happens, which saves the construction of string if the logger is configured to do nothing.